Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityController.cs

```diff
@@ -104,11 +105,27 @@ public void UpdateController(InteractionSourceState interactionSourceState)
         private void UpdateControllerData(InteractionSourceState interactionSourceState)
         {
             LastSourceStateReading = interactionSourceState;
+            var lastState = TrackingState;
 
-            isControllerTracked = interactionSourceState.sourcePose.TryGetPosition(out currentControllerPosition);
-            ControllerState = isControllerTracked ? ControllerState.Tracked : ControllerState.NotTracked;
+            switch (interactionSourceState.sourcePose.positionAccuracy)
```

Comment:
Recommend  removing this until we land the decision on #2347

Reply:
I disagree